packages/client/src/components/interface/DownloadButton.tsx

```diff
@@ -48,18 +48,16 @@ const StatusIndicator = styled.div<{
   isLoading?: boolean
 }>`
   display: flex;
-  flex-grow: 1;
+  flex-grow: 0;
   align-items: center;
   max-width: 152px;
   justify-content: ${({ isLoading }) =>
     isLoading ? `space-between` : `flex-end`};
 `
-const DownloadAction = styled(TertiaryButton)<{
-  isFullHeight?: boolean
-}>`
-  ${({ isFullHeight }) => isFullHeight && `height: 100%;`}
+const DownloadAction = styled(TertiaryButton)`
```

Comment:
Why is it TertiaryButton? Can't it be IconButton?

Reply:
@emon-swe-sust I think using TertiaryButton is dangerous as style may change there